### PR TITLE
Ruby & JavaScript (A very brief intro to CS): Enhancement of resource #28417

### PR DESCRIPTION
**Pull request by owen-makes**
## Because
Responding to issue https://github.com/TheOdinProject/curriculum/issues/28364

## This PR
1. In the [Ruby Course's Intro to CS Lesson](https://www.theodinproject.com/lessons/ruby-a-very-brief-intro-to-cs), add a new item in the Assignment section with this [video](https://www.youtube.com/watch?v=xWLxhF3b5P8).
2. In the [JavaScript Course's Intro to CS Lesson](https://www.theodinproject.com/lessons/javascript-a-very-brief-intro-to-cs), add this same item to the Assignment section

## Issue
Closes https://github.com/TheOdinProject/curriculum/issues/28364.

## Pull Request Requirements

-   [X] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [X] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [X] The `Because` section summarizes the reason for this PR
-   [X] The `This PR` section has a bullet point list describing the changes in this PR
-   [X] If this PR addresses an open issue, it is linked in the `Issue` section
-   [X] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [X] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/computer_science/a_very_brief_intro_to_cs.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ This section contains a general overview of topics that you will learn in this l
   1. Watch [What is an Algorithm?](https://youtu.be/e_WfC8HwVB8) on YouTube for a more structured look at solving problems using algorithms.
   1. Read this Quora answer about the [importance of algorithms in web development](https://qr.ae/py3NAc) to get some context for why we're going over this stuff.
   1. Watch [What is pseudocode?](https://www.youtube.com/watch?v=Rg-fO7rDsds)
+  1. Watch the first section (from 0:00 - 7:01) of [Telusko's video on Data Structure's and Algorithms](https://www.youtube.com/watch?v=xWLxhF3b5P8) for a quick overview of Data Structures and Algorithms and why companies may be interesting in hiring candidates familiar with DSA.
```
Comment (MaoShizhong) [inline javascript/computer_science/a_very_brief_intro_to_cs.md:28]:
```suggestion
  1. Watch the first section (from 0:00 - 7:01) of [Telusko's video on data structures and algorithms](https://www.youtube.com/watch?v=xWLxhF3b5P8) for a quick overview of DSA, and why companies may be interesting in hiring candidates familiar with DSA.
```

File: ruby/computer_science/a_very_brief_intro_to_cs.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ This section contains a general overview of topics that you will learn in this l
   1. Watch [What is an Algorithm?](https://youtu.be/e_WfC8HwVB8) on YouTube for a more structured look at solving problems using algorithms.
   1. Read this Quora answer about [the importance of algorithms in web development](https://qr.ae/py3NAc) to get some context for why we're going over this stuff.
   1. Watch [What is pseudocode?](https://www.youtube.com/watch?v=Rg-fO7rDsds)
+  1. Watch the first section (from 0:00 - 7:01) of [Telusko's video on Data Structure's and Algorithms](https://www.youtube.com/watch?v=xWLxhF3b5P8) for a quick overview of Data Structures and Algorithms and why companies may be interesting in hiring candidates familiar with DSA.
```
Comment (MaoShizhong) [inline ruby/computer_science/a_very_brief_intro_to_cs.md:28]:
```suggestion
  1. Watch the first section (from 0:00 - 7:01) of [Telusko's video on data structures and algorithms](https://www.youtube.com/watch?v=xWLxhF3b5P8) for a quick overview of DSA, and why companies may be interesting in hiring candidates familiar with DSA.
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Some nits below.

Please also note our contributing guide where we instruct [not to open a PR for an open issue unless you have been assigned by a maintainer](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md#being-assigned-an-issue). In this instance, I will assign you to the issue and accept this PR since no one else had requested assignment there. But in the future, we may not accept the PR if you've not been assigned, and will therefore close it without merging. Going forward, please comment on the relevant issue requesting assignment, then only open a PR if a maintainer has assigned it to you.

**Comment by owen-makes** [comment]
Thanks for letting me know! I was going through the "Using Git in the Real World" lesson and thought, why not take up this issue? I'll make sure request assignment before making a PR going forward :)

**Pull request merged**